packages/www/components/AssetDetails/AssetEventLogTab.tsx

```diff
@@ -30,7 +30,16 @@ const AssetEventLogTab = ({ asset }: { asset: Asset }) => {
               task.outputAssetId === asset.id || task.inputAssetId === asset.id
           )
           ?.map((task) => ({
-            type: task?.params?.export?.["ipfs"] ? "ipfs-upload" : "other",
+            type:
+              asset.source?.type == "url"
+                ? "url-import"
+                : asset.source?.type == "directUpload"
+                ? "file-import"
+                : asset.source?.type == "recording"
+                ? "recorded-session"
```

Comment:
The primary key on this UI is actually the `task` ("events"), not the asset, so I think this actually won't be equivalent. When we're showing data about an export task, we would actually show it as another entry for the asset upload instead. 

To fix you could surround these `asset.source?.type` checks under a `["upload", "import"].includes(task?.type)` condition instead, and only then you check the `asset`. WDYT?

Otherwise, can probably just replace `import` with `upload` on that previous implementation and let the `import` ones be ~brokenish.

Reply:
Ah.. yep. I thought there must've been something wrong with me doing that ha, yeah that fix by checking the task type makes sense 👍